.github/workflows/marker-report-issue.yml

```diff
@@ -0,0 +1,79 @@
+name: Open marker report GH issue
+on:
+  # only manual for now
+  workflow_dispatch:
+  pull_request:  # TODO: remove before merge
+    branches:
+      - master
+
+jobs:
+  marker-report-issue:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+
+      - name: Checkout this repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        id: setup-python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Cache LocalStack community dependencies (venv)
+        uses: actions/cache@v3
+        with:
+          path: .venv
+          # we can re-use the venv from other actions, but shouldn't write one that may be used by others since we install additional deps
+          key: ${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('setup.cfg') }}-marker-report-issue
+          restore-keys: |
+            ${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('setup.cfg') }}
```

Comment:
Is this key used anywhere else / is there the possibility that there is a cache hit on this key?

Reply:
The thing is this will be run on master, so if I understand correctly it could overwrite the cached venv from the pro integration tests against community for example. (given that the keys overlap).

Reply:
No, I mean the restore-key. Is this one set anywhere? 
If it is: Which cache is it you intend to reuse here?
If not: What's the purpose of the restore key here then?

Reply:
Not sure why but I thought this was matching the pro integration tests, which doesn't seem to be the case. It's also not actually that important to cache at all here. Given the intended long period between different runs of this action, it seems it's probably clearer to just remove the cache completely